azure-pipelines.vrt-baseline.yml

```diff
@@ -20,27 +20,23 @@ jobs:
       clean: all
```

Comment:
## [🕵 fluentuiv8 Open the Visual Regressions report to inspect the 3 screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-fluent/index.html?organization=uifabric&project=fabricpublic&prId=28294&commitId=3d04ac16975889be0ec211b943e03c44c0b6e553&env=prod)

✅ There was 0 screenshots added, 0 screenshots removed, 1044 screenshots unchanged, 0 screenshots with different dimensions and 3 screenshots with visible difference.


  <details><summary><b>unknown</b> 3 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|ComboBox.Root.click.chromium.png|13| Changed|
|ComboBox.Root.default.chromium.png|10| Changed|
|ComboBox.Root.hover.chromium.png|10| Changed|

</details>


<div id="vrtCommentfluentuiv8"/>